showcase/src/app/demo-ui/pages/date-range-picker/demo-date-range-picker-page.component.ts

```diff
@@ -33,12 +41,19 @@ export class DemoDateRangePickerPageComponent {
 		}
 	];
 
-	public constructor(@Inject(STARK_LOGGING_SERVICE) public logger: StarkLoggingService) {}
+	public constructor(@Inject(STARK_LOGGING_SERVICE) public logger: StarkLoggingService) {
+		this._subs.push(this.dateRangeFormGroup.valueChanges.subscribe((v: any) => this.logger.debug(v)));
+	}
 
-	public onDateChanged(event: StarkDateRangePickerEvent): void {
-		this.logger.debug(event);
+	public onDateModelChange(): void {
+		this.logger.debug(this.dateRangeModel);
+	}
+
+	public onDateChange(dateRange: StarkDateRangePickerEvent): void {
+		this.logger.debug(dateRange);
```

Comment:
Can you add some text to this logging to know what is being logged? Otherwise is confusing in the browser console...